Common/Source/LKInterface/LKCustomKeyHandler.cpp

```diff
@@ -320,7 +320,10 @@ bool CustomKeyHandler(const int key) {
 		PlayResource(TEXT("IDR_WAV_CLICK"));
 		InputEvents::eventWaypointDetails(_T("select"));
 		return true;
-
+    case ckAirspaceLookup:
```

Comment:
case must be ordered

Reply:
???

Reply:
writing

switch(..) {
case 1:
case 2:
case 3:
}

are better than writing

switch(..) {
case 3:
case 1:
case 2:
}

especially when all cases are contiguous value